src/shared/components/what-can-you-build-section/accordion/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import itemsArr from 'shared/data/what-can-you-build';
+import AccordionItem from 'shared/components/what-can-you-build-section/accordion/accordion-item';
+import styles from './index.module.css';
+
+class Accordion extends Component {
+    constructor() {
+        super();
+
+        this.state = {
+            openedItemIndex: -1,
+        };
+
+        this.handleAccordionItemClick = this.handleAccordionItemClick.bind(this);
+    }
+
+    render() {
+        const { className } = this.props;
+        const accordionClasses = classNames(styles.accordion, className);
+        const { openedItemIndex } = this.state;
+        const items = itemsArr.map((item, index) => {
+            const isOpened = openedItemIndex === index;
+
+            return (
+                <AccordionItem
+                    key={ `accordionItem-${index}` }
+                    item={ item }
+                    index={ index }
+                    onClick={ this.handleAccordionItemClick }
+                    isOpened={ isOpened } />
+            );
+        });
+
+        return (
+            <div className={ accordionClasses } >
+                { items }
+            </div>
+        );
+    }
+
+    handleAccordionItemClick(accordionItemIndex) {
+        this.setState(({ openedItemIndex }) => ({ openedItemIndex: accordionItemIndex === openedItemIndex ? -1 : accordionItemIndex }));
```

Comment:
As this line has some logic with the ternary condition, I think that it is cleaner as follows:

```js
this.setState(({ openedItemIndex }) => ({ 
    openedItemIndex: accordionItemIndex === openedItemIndex ? -1 : accordionItemIndex 
}));
```